target/linux/ramips/mt7620/base-files/etc/board.d/01_leds

```diff
@@ -148,6 +148,14 @@ lenovo,newifi-y1s)
 	ucidef_set_led_netdev "wifi5g" "WIFI5G" "blue:wifi" "wlan0"
 	ucidef_set_led_netdev "wan" "WAN" "blue:internet" "eth0.2" "tx rx"
 	;;
+netcore,nw5212)
+	ucidef_set_led_switch "wan" "wan" "$boardname:green:wan" "switch0" "0x10"
+	ucidef_set_led_switch "lan1" "lan1" "$boardname:green:lan1" "switch0" "0x08"
+	ucidef_set_led_switch "lan2" "lan2" "$boardname:green:lan2" "switch0" "0x04"
+	ucidef_set_led_switch "lan3" "lan3" "$boardname:green:lan3" "switch0" "0x02"
+	ucidef_set_led_switch "lan4" "lan4" "$boardname:green:lan4" "switch0" "0x01"
+	ucidef_set_led_wlan "wlan2g" "wlan2g" "$boardname:green:wlan" "phy0tpt"
```

Comment:
WLAN line can be removed here since you have the DT trigger.

After that, you can probably just merge this case with another one ...